cpp/farm_ng/core/prototools/event_log_reader.h

```diff
@@ -62,7 +91,26 @@ class EventLogReader {
   void reset();
 
  private:
-  std::unique_ptr<EventLogReaderImpl> impl_;
+  std::shared_ptr<EventLogReaderImpl> impl_;
 };
 
+core::proto::Timestamp const* getStamp(
```

Comment:
returns pointer, a line of comment what the semantic is (e.g. no memory allocation, n/a stamp -> nullptr)